images-list

```diff
@@ -220,6 +220,7 @@ gcr.io/google_containers/k8s-dns-sidecar rancher/mirrored-k8s-dns-sidecar 1.15.1
 gcr.io/google_containers/pause rancher/mirrored-pause 3.1
 gcr.io/google_containers/pause rancher/mirrored-pause 3.2
 gcr.io/kubebuilder/kube-rbac-proxy rancher/kube-rbac-proxy v0.5.0
+gcr.io/kubebuilder/kube-rbac-proxy rancher/mirrored-kubebuilder-kube-rbac-proxy v0.18.0
```

Comment:
why are we changing the target from `rancher/mirrored-kube-rbac-proxy` to `rancher/mirrored-kubebuilder-kube-rbac-proxy`? I think the target should remain as is, and v0.18.0 should go at the bottom of the list just below this line.

If the target change is intentional, can you confirm that the new repo exists and the bot account can push to it?

Reply:
Your right, didn't catch this @brandond